docs/source/release/v6.4.0/index.rst

```diff
@@ -4,19 +4,51 @@
 Mantid 6.4.0 Release Notes
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../images/SliceViewer_CutViewer.png
    :class: screenshot
    :width: 385px
    :align: right
 
 .. contents:: Table of Contents
    :local:
 
-.. warning:: This release is still under construction. The changes can be found in the nightly builds on the `download page`_.
-
 We are proud to announce version 6.4.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+Many of the updates in this version have focused on improvements for users, building on existing algorithms and interfaces.
+Our method of packaging has also changed. Windows and macOS users should not notice any difference,
+but this will be the last release that uses the linux ``.rpm`` and ``.deb`` packages (more details :ref:`here <6.4_packaging>`).
```

Comment:
This is slightly picky but could we reword this to: 

_but this will be the last release that we provide ``.rpm`` and ``.deb`` packages for Linux (more details :ref:`here <6.4_packaging>`)._